imports/ui/components/study_plan/ModuleSearchBar.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import Autosuggest from 'react-autosuggest';
+
+export default class Module extends React.Component {
```

Comment:
is this 'export default class Module extends React.Component'
or
'export default class ModuleSearchBar extends React.Component?